content/guides/models/app/features/panels/_index.md

```diff
@@ -125,6 +151,13 @@ To add a section, click **Add section** after the last section.
 
 To add a new section before or after an existing section, you can instead click the section's action `...` menu, then click **New section below** or **New section above**.
 
+
+### Manage a section's visible panels
+By default, each section shows 6 panels. To customize a section that has more than 6 panels:
+
+1. At the top of the section, click **1 to 6 of <X>**, where `<X>` is the total number of panels.
+1. Choose how many panels to show, or click **Show all panels**.
```

Comment:
i believe we enforce a max page size of 100 panels. "show all" will only be available if num panels in section < that max

i don't think it's necessary to explicitly call out the "show all" option.

```suggestion
1. Choose how many panels to show per page.
```

Reply:
Done!